packages/create-fuels/src/cli.ts

```diff
@@ -130,26 +139,40 @@ export const runScaffoldCli = async ({
 }) => {
   program.parse(args);
 
-  const projectPath = program.args[0] ?? (await promptForProjectPath());
+  let projectPath = program.args[0] ?? (await promptForProjectPath());
   const verboseEnabled = program.opts().verbose ?? false;
 
-  if (existsSync(projectPath)) {
-    throw new Error(
-      `A folder already exists at ${projectPath}. Please choose a different project name.`
+  while (existsSync(projectPath)) {
+    log(
+      chalk.red(
+        `A folder already exists at ${projectPath}. Please choose a different project name.`
+      )
     );
+
+    // Exit the program if we are testing to prevent hanging
+    if (process.env.VITEST) {
+      throw new Error();
+    }
```

Comment:
```suggestion
    /* istanbul ignore next -- @preserve */
    if (process.env.VITEST) {
      // Exit the program if we are testing to prevent hanging
      throw new Error();
    }
```

Reply:
@danielbate So, as I said [here](https://github.com/FuelLabs/fuels-ts/pull/2014#discussion_r1554583407), the Istanbul ignore unnecessary in this case because tests _will_ run it. 🙂